Source/Fuse.Common/Internal/SystemFont.uno

```diff
@@ -11,6 +11,13 @@ namespace Fuse.Internal
 		public readonly int Index;
 		public readonly IEnumerable<string> Styles;
 
+		//iOS13+ - for default systemfont
+		public bool IsCustomFont = true;
```

Comment:
Are these specific for iOS? If so, we can do `extern(IOS) bool IsCustomFont = ...`.

Also, can we drop `public`? Or make `internal` instead?

Reply:
`extern(iOS)` breaks compilation, was able to make it internal